msteams-platform/tutorials/get-started-yeoman.md

```diff
@@ -129,4 +138,39 @@ Choose your tab and follow the instructions to add it. Notice that you have a cu
 
 ![running tab in teams](~/assets/yeoman-images/teams-first-app-6.png)
 
-**Congrats! You built and deployed your first Microsoft Teams app**
+## Upgrade Microsoft Teams
```

Comment:
@KirtiPereira Docs validation tool generates a warning if you do not have a space between a header and body text (or other element).